test/get-resource-id.js

```diff
@@ -38,7 +39,7 @@ describe("Testing jsonapi-server", function() {
 
     it("with fields", function(done) {
       var url = "http://localhost:16006/rest/articles/de305d54-75b4-431b-adb2-eb6b9e546014?fields[articles]=title";
-      helpers.request({
+      request({
```

Comment:
`helpers.request` is a wrapper around `request` that also validates the request + response against the swagger definition. The `fields` JSON:API attribute allows us to bypass the required attributes on the models, so I've had to opt a bunch of these tests out of those assertions.